uint8 cmdID
uint8 errorCode
float64 Latitude
float64 Longitude